packages/ui/src/universal/error-boundary/index.test.tsx

```
@@ -0,0 +1,81 @@
+import { describe, it, expect, vi, beforeEach, afterEach } from 'vitest';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { ErrorBoundary, ErrorFallback } from '.';
+
+describe('ErrorBoundary', () => {
+  beforeEach(() => {
+    vi.spyOn(console, 'error').mockImplementation(() => {});
+    // Mock window.location.reload
+    const reloadMock = vi.fn();
+    const originalLocation = window.location;
+    // @ts-ignore mock implementation
+    delete window.location;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)